tests/Platforms/AcquiaTest.php

```
@@ -77,134 +78,86 @@ function ($test): void {
-    global $settings;
-    global $config;
+    $settings = $initial_settings;
+    $config = [];
+    $context = new Drupal($settings, $config);
+    (new Acquia())->contextualize($context);
 
-    $this->assertEquals($expected['settings'], $settings);
-    $this->assertEquals($expected['config'], $config);
+    $this->assertEquals($expected_settings, $settings);
+    $this->assertEquals($expected_config, $config);
   }
```

PHPMD (2.15.0)

[error] The parameter $initial_settings is not named in camelCase. (undefined)

(CamelCaseParameterName)